python/hail/matrixtable.py

```diff
@@ -614,9 +604,30 @@ def row(self):
         """
         return self._row
 
+    @property
+    def row_value(self):
+        """Returns a struct expression including all non-key row-indexed fields.
+
+        Examples
+        --------
+
+        Get the first five non-key row field names:
+
+        .. doctest::
```

Comment:
I've learned you don't actually need to enclose this in a doctest block. You can delete the directive and bring the two lines below back a tab.